pkg/cmd/util/clientcmd/factory.go

```diff
@@ -122,9 +123,12 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 		}
 		return deployreaper.ReaperFor(mapping.Kind, oc, kc)
 	}
+	kGeneratorFunc := w.Factory.Generator
 	w.Generator = func(name string) (kubectl.Generator, bool) {
-		generator, ok := generators[name]
-		return generator, ok
+		if generator, ok := generators[name]; ok {
+			return generator, true
+		}
+		return kGeneratorFunc(name)
```

Comment:
Since we fallback to Kubernetes, we don't need "service/v1" in our own generators. L62 - remove it please.